cibuildwheel/docker_container.py

```diff
@@ -31,7 +34,11 @@ class DockerContainer:
     bash_stdout: IO[bytes]
 
     def __init__(
-        self, *, docker_image: str, simulate_32_bit: bool = False, cwd: Optional[PathOrStr] = None
+        self,
+        *,
+        docker_image: str,
+        simulate_32_bit: bool = False,
+        cwd: Optional[PathOrStr] = None,
```

Comment:
Can you restore this as it was before ?
I guess it was split over multiple lines when the fix went through an argument to this function.